doc/releases/release-notes-3.7.rst

```diff
@@ -504,11 +504,75 @@ Drivers and Sensors
 
 * Ethernet
 
-  * Deprecated eth_mcux driver in favor of the reworked nxp_enet driver.
-  * Driver nxp_enet is no longer experimental.
-  * All boards and SOCs with :dtcompatible:`nxp,kinetis-ethernet` compatible nodes
-    reworked to use the new :dtcompatible:`nxp,enet` binding.
-  * Added support for PTP on compatible STM32 series (STM32F7, STM32H5 and STM32H7).
+  * Introduced :kconfig:option:`CONFIG_ETH_DRIVER_RAW_MODE`. This option allows building
+    ethernet drivers without the zephyr L2 ethernet layer.
+  * Removed the ethernet-fixed-link DT binding.
+  * Removed VLAN handling from ethernet drivers since it is now handled by the
+    generic ethernet L2 code.
+  * Implemented/reworked HW MAC Address filtering in the eth_mcux, eth_nxp_enet,
+    and eth_nxp_s32_gmac, eth_stm32, and eth_nxp_s32_netc drivers.
+  * New Drivers
+     * Added new eth_nxp_enet_qos driver for the ethernet controller present on NXP MCXN SOCs.
+     * Added support for adin1100 phy.
+     * Added support for the Realtek RTL8211F phy.
+  * NXP ENET driver changes
```

Comment:
Empty line needed below this

Reply:
can you explain to me why this is needed ?

Reply:
So it renders as a nested list just like the other. Isn't that what you were going for?
(you can see the render of this PR here:
https://builds.zephyrproject.io/zephyr/pr/75001/docs/releases/release-notes-3.7.html
)

Reply:
nevermind, I think I see why, there is a highlight for some reason with the 3 space indent in zephyr docs

Reply:
woops, your comment didn't load for me until now